### PR TITLE
R4R:add new fields for proposal detail api

### PR DESCRIPTION
**Pull request by wenwu-bianjie**
https://github.com/irisnet/explorer/issues/1033

### CODE REVIEW

File: backend/vo/proposal.go [hunk 1]
```diff
@@ -51,6 +51,13 @@ type Proposal struct {
 	Software        string      `json:"software,omitempty"`
 	SwitchHeight    uint64      `json:"switch_height,omitempty"`
 	Threshold       string      `json:"threshold,omitempty"`
+	Level           string      `json:"level"`
+	YesThreshold    string      `json:"yes_threshold"`
+	VetoThreshold   string      `json:"veto_threshold"`
+	Participation   string      `json:"participation"`
+	Penalty         string      `json:"penalty"`
+	Usage           string      `json:"usage,omitempty"`
```
Comment (kaifei-bianjie) [inline backend/vo/proposal.go]: delete flag `omitempty `

File: backend/service/service_proposal.go [hunk 2]
```diff
@@ -588,6 +599,53 @@ func (service *ProposalService) Query(id int) (resp vo.ProposalInfoVo) {
...
+
+		burnPercent, err := lcd.GetProposalBurnPercentByResult(data.Status, isRejectVote)
+		if err != nil {
+			logger.Error("GetProposalBurnPercentByResult fail", logger.String("err", err.Error()))
+		} else {
+			proposal.BurnPercent = burnPercent
+		}
 	}
 
 	from, to, err := document.Proposal{}.QueryTxFromToByTypeAndProposalId(id)
```
Comment (kaifei-bianjie) [inline backend/service/service_proposal.go]: proposal.TxHash = to??

File: backend/vo/proposal.go (hunk 1 above)
Comment (wenwu-bianjie) [reply backend/vo/proposal.go]: ok

File: backend/service/service_proposal.go (hunk 2 above)
Comment (wenwu-bianjie) [reply backend/service/service_proposal.go]: change 'to' to 'hash'

**Pull request merged**